Projects/TEAM-2/Tsvihun/Task_301/Shop.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Task_301
+{
+    internal class Shop
+    {
+        private List<Goods> goods = new List<Goods>();
+
+        public void AddItem(string name, int price, int quantity)
+        {
+            bool itemIsPresent = false;
+            for (int i = 0; i < goods.Count; i++)
+            {
+                if (name == goods[i].name)
+                {
+                    itemIsPresent = true;
+                    goods[i].quantity += quantity;
+                    break;
+                }
+            }
+            if (!itemIsPresent)
+            {
+                goods.Add(new Goods(name, price, quantity));
+            }
+            Console.WriteLine($"Added {quantity} of {name} !");
+        }
+
+        public void SellItem(string name, int quantity)
+        {
+            //1 - Find item by name. Remove specified amount of them
+            for (int i = 0; i < goods.Count; i++)
+            {
+                if (name == goods[i].name)
+                {
+                    goods[i].quantity -= quantity;
```

Comment:
It seems like we are able to sell goods even in the case goods[i].quantity = 0

So it could be better to add the validation for quantity parameter

Reply:
Made it.